git-flow/PKGBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: Johannes Schindelin <johannes.schindelin@gmx.de>
+
+_realname=git-flow
+pkgname=("${_realname}")
+pkgver=0.4.1.108.g15aab26
+pkgrel=1
+pkgdesc="Git extensions to provide high-level repository operations for Vincent Driessen's branching model (mingw-w64)"
```

Comment:
Whoopsie, sorry! Fixed.

For background: I had originally played with the idea of making `git-flow` a MinGW package depending on MinGW Git, but decided against it due to its use of the `getopt.exe` file of `util-linux`.